autoload/go/coverage.vim

```diff
@@ -46,7 +46,7 @@ function! go#coverage#Buffer(bang, ...) abort
 
   if go#util#has_job()
     call s:coverage_job({
-          \ 'cmd': ['go', 'test', '-coverprofile', l:tmpname],
+          \ 'cmd': ['go', 'test', '-coverprofile', l:tmpname] + a:000,
```

Comment:
Visually it looks the same as :GoCoverage -- it will instead only show lines covered by the particular test(s) run.